include/kernels/clover_outer_product.cuh

```diff
@@ -30,11 +32,17 @@ namespace quda {
     int displacement;
     bool partitioned[4];
     real coeff;
+    const unsigned int volume_4d_cb;
+    const unsigned int GhostFace_4d_cb;
 
     CloverForceArg(GaugeField &force, const GaugeField &U, const ColorSpinorField &inA, const ColorSpinorField &inB,
                    const ColorSpinorField &inC, const ColorSpinorField &inD, const unsigned int parity,
                    const double coeff) :
-      kernel_param(dim3(dim == -1 ? inA.VolumeCB() : inB.GhostFaceCB()[dim])), // inB since it has a ghost allocated
+      // kernel_param(dim3(dim == -1 ? inA.VolumeCB() : inB.GhostFaceCB()[dim])), // inB since it has a ghost allocated
```

Comment:
This comment can be deleted

Reply:
7acf254ba019063551721afd3969ab77469eb8a0